data/faq.md

```diff
@@ -93,18 +118,22 @@ A. Yes, our
 
 A. The time and location of the conference have been set, so you can book at any time. We would recommend you consider flexible bookings in case of any changes of plans in the future as well as appropriate insurance cover.
 
+## **Q. Could you provide a visa invitation letter for my visa application?**
+
+A. Yes. We will set up a dedicated page with details on how to obtain the invitation letter in mid April. Please refer to the official [Irish Immigration Service](https://www.irishimmigration.ie/) for information, including the application process for an [Irish Conference/Event Visa](https://www.irishimmigration.ie/coming-to-work-in-ireland/what-are-my-options-for-working-in-ireland/coming-to-work-for-less-than-90-days/conference-event-visa/).
+
+
 ## **Q. Are you going to offer hotel discounts?**
 
-A. As of now there’s no discount that we know of or can offer. Make sure to subscribe to our [EuroPython Blog](https://blog.europython.eu/#/portal/signup) to stay tuned.
+A. As of now there’s no discount that we know of or can offer. We hope to give you suggestions in Mid-late April. Make sure to subscribe to our [EuroPython Blog](https://blog.europython.eu/#/portal/signup) to stay tuned.
 
 ## **Q. When can sponsors start signing up?**
 
-A. We are aiming to start sponsor signup in late March. Please check
-[2019 Sponsor Packages](https://ep2019.europython.eu/sponsor/packages/) to get an idea of the sponsorship options. The packages are not finalised yet, but will be similar to what we had in 2019. Sponsors who sign up early will again receive a 10% discount.
+A. Sponsors are welcome to sign up now! You can find all the details on our [sponsor page](https://ep2022.europython.eu/sponsor). Sponsors who sign up by 29 April will again receive a 10% Early Bird discount.
 
-We are committed to making EuroPython sponsorship accessible to most organisations keen to help the community.
+We are committed to making EuroPython sponsorship accessible to most organisations keen to help the community. [SME discount](https://ep2022.europython.eu/sponsor#sme) and [Free Table for Open Source Projects](https://ep2022.europython.eu/sponsor#free-table-for-open-source-projects) are offerings driven by our community spirit.
```

Comment:
"accessible to as many organisations as possible, who are keen to help the community."

Reply:
Big yes!